integration/utils/contLoadGenerator.go

```diff
@@ -380,6 +455,125 @@ func (lg *ContLoadGenerator) sendAddKeyTx(workerID int) {
 	<-ch
 }
 
+func (lg *ContLoadGenerator) addKeysToProposerAccount(proposerPayerAccount *flowAccount) error {
+	if proposerPayerAccount == nil {
+		return errors.New("proposerPayerAccount is nil")
+	}
+
+	addKeysToPayerTx, err := lg.createAddKeyTx(*lg.accounts[0].address, lg.constExecParam.PayerKeyCount)
+	if err != nil {
+		lg.log.Error().Msg("failed to create add-key transaction for const-exec")
+		return err
+	}
+	addKeysToPayerTx.SetReferenceBlockID(lg.follower.BlockID()).
+		SetProposalKey(*lg.accounts[0].address, 0, lg.accounts[0].seqNumber).
+		SetPayer(*lg.accounts[0].address)
+
+	lg.log.Info().Msg("signing the add-key transaction for const-exec")
+	err = lg.accounts[0].signTx(addKeysToPayerTx, 0)
+	if err != nil {
+		lg.log.Error().Err(err).Msg("error signing the add-key transaction for const-exec")
+		return err
+	}
+
+	lg.log.Info().Msg("issuing the add-key transaction for const-exec")
+	ch, err := lg.sendTx(0, addKeysToPayerTx)
+	if err != nil {
+		return err
+	}
+	<-ch
+
+	lg.log.Info().Msg("the add-key transaction for const-exec is done")
+	return nil
+}
+
+func (lg *ContLoadGenerator) getTxSizeWithoutCommnent() uint {
```

Comment:
maybe rename to getConstExecCostTxSizeWithoutComment?

Reply:
done